### PR TITLE
add banner for projects where whole team from Ukraine. refs #120

### PR DESCRIPTION
**Pull request by dmitry-sinina**

### CODE REVIEW

**Comment by webknjaz** [comment]
I believe that this needs to be wired to the templating setup we have now. As for the wording, it seems grammatically correct, no objections for me.

**Comment by dmitry-sinina** [comment]
>I believe that this needs to be wired to the templating setup we have now.

What templating you saying about?

**Comment by webknjaz** [comment]
https://github.com/vshymanskyy/StandWithUkraine/blob/d1f0c46/generate.js#L55-L59

cc @mourner

File: banner-direct-team.svg [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  }
+}
+  </style>
+
+  <a href="https://stand-with-ukraine.pp.ua">
+    <rect x="0" y="0" width="100%" height="100%" class="flag-yellow"/>
+    <rect x="0" y="0" width="100%" height="110px" class="flag-blue"/>
+
+    <text x="0" y="25" class="message">
+      <tspan x="25" dy="0.45em">This project is created by the team from Ukraine.</tspan>
```
Comment (mourner) [inline banner-direct-team.svg:51]: The team -> a team

**Review by mourner — CHANGES_REQUESTED** [review]
A grammar nit aside from templating

**Review by mourner — APPROVED** [review]

**Pull request merged**